utils/utilFunctions.ts

```diff
@@ -0,0 +1,18 @@
+// THESE ARE USED IN DEVELOPMENT MOSTLY
+
+// CHECK THIS, kehittämisvaiheessa käytettäviä
+
+// miten toimii, jos tietokannalla kestää vaikka 5 sec palauttaa vastaus ym
+
+export async function sleep(timeoutTimeInMs: number) {
+  await new Promise((r) => setTimeout(r, timeoutTimeInMs));
+}
+
+/**
+ *
+ * @returns either false or true
+ */
+
+export function randomBoolean() {
+  return !Math.round(Math.random());
+}
```

Comment:
Tein uuden tiedoston nimellä `utilFunctions.ts`. Tässä on vaan se, että nämä ovat ehkä lähempänä devaus funktioita. Eli ettei käytetä oikeassa koodissa.

Ehkä jokin `developmentFunctions` nimi voisi olla parempi :thinking: 

`randomBoolean`in tarkoitus on vain palauttaa joko True tai False. Käytin sitä testatakseni Tanstackin `isError`ia. Eli arvoin true vai false ja sen perusteella palautin joko virheen tai sitten oikean tiedon

Reply:
Tiedoston nimi on hyvä. Vaikka näitä funktioita nyt käytetään vain testaukseen, niin ne toimivat nimensä mukaisesti ja etenkin `sleep` funktiolle saattaa hyvin löytyä käyttöä oikeasta koodista.

Reply:
> Tiedoston nimi on hyvä. Vaikka näitä funktioita nyt käytetään vain testaukseen, niin ne toimivat nimensä mukaisesti ja etenkin `sleep` funktiolle saattaa hyvin löytyä käyttöä oikeasta koodista.

Onko `utils`-kansio hyvä vai pitäisikö siirtää `shared`iin? Siellä ei tällä hetkellä ole mitään funktioita, mitä tarvittaisiin backendissä, joten sinällään ehkä itse pitäisin `utils`issa